src/kvstore/kvstore_dist_server.h

```diff
@@ -125,6 +127,8 @@ class KVStoreDistServer {
   }
 
   ~KVStoreDistServer() {
+    profiler::Profiler::Get()->SetState(
+      profiler::Profiler::ProfilerState(profiler::Profiler::kNotRunning));
     delete ps_server_;
```

Comment:
@cjolivier01 Is this required?

Reply:
not sure what the context is when this is called. is this an independent process? normally profiler is shut down by static shutdown, after engine